packages/contracts/test/router/v6/sudoswap/offers.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Contract } from "@ethersproject/contracts";
+import { parseEther } from "@ethersproject/units";
+import * as Sdk from "../../../../../sdk/src";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-with-address";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { ExecutionInfo } from "../../helpers/router";
+import {
+  getChainId
+} from "../../../utils";
+
+describe("[ReservoirV6_0_0] Sudoswap offers", () => {
+  const chainId = getChainId();
+
+  let deployer: SignerWithAddress;
+  let alice: SignerWithAddress;
+  let bob: SignerWithAddress;
+  let carol: SignerWithAddress;
+  let david: SignerWithAddress;
+  let emilio: SignerWithAddress;
+
+  let router: Contract;
+  let sudoswapModule: Contract;
+
+  beforeEach(async () => {
+    [deployer, alice, bob, carol, david, emilio] = await ethers.getSigners();
+
+    router = (await ethers
+      .getContractFactory("ReservoirV6_0_0", deployer)
+      .then((factory) => factory.deploy())) as any;
+    sudoswapModule = (await ethers
+      .getContractFactory("SudoswapModule", deployer)
+      .then((factory) =>
+        factory.deploy(router.address, router.address, Sdk.Sudoswap.Addresses.PairRouter[chainId])
+      )) as any;
+  });
+
+  /**
+   * npx hardhat test test/router/v6/sudoswap/offers.test.ts
+   */
+  it("Sudoswap router test", async () => {
+    // Setup
+
+    let tokenId = 6113;
+
+    let addresPDB = "0xaCd1423E1e7D45DD0F3AE63C5dB959D49FeADd3F";
+    let abiOwnerOf = '[{"inputs":[{"internalType":"uint256","name":"tokenId","type":"uint256"}],"name":"ownerOf","outputs":[{"internalType":"address","name":"","type":"address"}],"stateMutability":"view","type":"function"}]';
+    let contractPDB = new ethers.Contract(addresPDB, abiOwnerOf, ethers.provider);
```

Comment:
Could use the helpers here to avoid hardcoding ABIs.

https://github.com/reservoirprotocol/core/blob/059a6b5f2537be537871fb6febc531c9cf3870f0/packages/contracts/test/sdk/looks-rare/single-token/erc721.test.ts#L50